packages/richtext-lexical/src/features/toolbars/shared/ToolbarButton/index.scss

```diff
@@ -18,7 +18,7 @@
     margin-right: 2px;
   }
 
-  &:hover:not([disabled]) {
+  &:hover:not(.disabled) {
```

Comment:
I realized that this wasn't working. Individual buttons allowed hover even when disabled (see example with indent). The reason is that disabled appears there as a class